docs/development/developing-dapr.md

```diff
@@ -1,81 +1,12 @@
 # Developing Dapr
 
-This section will walk you through on how to clone and build the Dapr runtime.
-First, make sure you have [Go 1.16](https://golang.org/dl/) installed.
+## Setup Dapr development environment
 
+There are several options for getting an environment up and running for Dapr development:
 
-## Setup environment
-
-This document helps you get started developing Dapr. If you find any problem while following this guide, please create a Pull Request to update this document.
-
-### Docker environment
-
-1. Install [Docker](https://docs.docker.com/install/)
-    > For Linux, you'll have to configure docker to run without sudo for this to work, because of the environment variables.  See the following on how to configure [this](https://docs.docker.com/install/linux/linux-postinstall/).
-
-2. Create your [Docker Hub account](https://hub.docker.com)
-
-
-### Go dev environment
-
-#### Linux and MacOS
-
-1. The Go language environment [(instructions)](https://golang.org/doc/install#tarball).
-   * Make sure that your GOPATH and PATH are configured correctly
-   ```bash
-   export GOPATH=~/go
-   export PATH=$PATH:$GOPATH/bin
-   ```
-2. [Delve](https://github.com/go-delve/delve/tree/master/Documentation/installation) for Debugging
-
-#### Windows
```

Comment:
I actually use this configuration. Why are we deleting these instructions?

Reply:
They are duplicated in the setup-dapr-development-env.md, with slight variations that caused the confusion in #2529. This PR just consolidates these instructions into setup-dapr-development-env.md and removes the duplication in developing-dapr.md, which now points to the other doc for manual install steps.